db/db_client/db_client_execute.go

```diff
@@ -75,6 +75,10 @@ func (c *DbClient) Execute(ctx context.Context, query string, disableSpinner boo
 	return c.ExecuteInSession(ctx, sessionResult.Session, query, closeSessionCallback, disableSpinner)
 }
 
+// ExecuteInSession implements Client
+// execute the provided query against the Service in the given `context.Context` using the provided `*DatabaseSession`
```

Comment:
no slanted quotes in comments 
```
// ExecuteInSession implements Client
// execute the query in the given Context using the provided DatabaseSession
// ExecuteInSession assumes no responsibility over the lifecycle of the DatabaseSession - that is the responsibility of the caller
// NOTE: The returned Result MUST be fully read - otherwise the connection will block and will prevent further communication